docs/source/markdown/podman-manifest-push.1.md.in

```diff
@@ -42,6 +42,12 @@ the list or index itself. (Default true)
 
 @@option digestfile
 
+#### **--force-compression**
```

Comment:
Note please do not duplicate the same text in two man pages, put it in markdown/options and use the @@option import syntax like the other flags for push and manifest push.